### DISCUSSION TITLE
Ping feature?

### DISCUSSION THREAD

**Discussion by pickles4evaaaa** [henrygd/beszel#91] [Ideas]
Beszel has been great, thank you! I wanted to suggest a ping feature, similar to other monitor applications, where you can check the status of a LAN device by simply sending it a ping. This may be outside the point/use case of beszel, but I would like to completely move to beszel instead of having uptime kuma for the ping feature. Just an idea :)

**Comment by TerenceS79** [comment]
yes please - a simple up/down monitor (using ICMP ping) would be perfect. some of my devices don't support Beszel agent installs.

**Comment by hiphiparrays** [comment]
Adding another vote for ping 🙂

**Comment by That-Dude** [comment]
I thought I was missing something, so there's no way to monitor a router / switch / network printer with Beszel?

What are you guys using for this functionality?

**Reply by henrygd** [reply]
Gatus and Uptime Kuma are fairly popular. We'll likely add basic functionality for this at some point.

**Reply by That-Dude** [reply]
Thanks Henry, I'll give them shot!
